### ISSUE TITLE
Translation instructions for iD are unclear (unexpected question when signing up for Transifex)

### ISSUE THREAD

**Issue by mnalis** [openstreetmap/iD#9562]
https://github.com/openstreetmap/iD/blob/develop/CONTRIBUTING.md#translating gives instructions to:
> Translations are managed using the [Transifex](https://www.transifex.com/openstreetmap/id-editor/) platform. After signing up, you can go to [iD's project page](https://www.transifex.com/openstreetmap/id-editor/), select a language and click Translate to start translating

However, when clicking on `Sign up` on Transifex, after entering username/email/password, user is asked whether to start free trial (which will supposedly create new team and expire in 15 days?) or if they want to join existing organization (in which case one supposedly needs to select which organization). . . 

**Instructions should specify which of those should be chosen** (and what extra info needs to be inputted there, if any)

![Screenshot 2023-03-25 at 13-40-34 Transifex Localization Platform](https://user-images.githubusercontent.com/156656/227718276-b79d91b1-b35d-487c-82c0-d99be0e7fa14.png)

**Comment by matkoniecz** [comment]
join an existing organization work well, right?

I guess that is a good sign to use something else for new projects

**Comment by mnalis** [comment]
> join an existing organization work well, right?

I'll tell you in 15 days :smile: 

I've chosen `Join an existing organization` and then opened authorization link from e-mail, and after that opened next link mentioned in docs: https://www.transifex.com/openstreetmap/id-editor/ and chosen languages I use. 

It seems to have let me in and allows me to edit Croatian iD texts, but I don't really know if it assigned me an organization or not.

> I guess that is a good sign to use something else for new projects

Yeah I usually beware of such sites offering "free trials" with expiration dates. In my experience, they always end up removing more and more features from "free" tier until it becomes unusable and then projects have to either waste time and effort breaking out of [vendor lock-in](https://en.wikipedia.org/wiki/Vendor_lock-in), or give up and start paying in order to stop such abuse and continue using what they were using. 
But I'm no decision maker for iD :man_shrugging: , nor do I have enough background information to make informed decision even if I were.

**Comment by matkoniecz** [comment]
> I'll tell you in 15 days 😄

@mnalis

**Comment by mnalis** [comment]
>  >  join an existing organization work well, right?

> I'll tell you in 15 days 😄

It seems so (i.e. I am member nowadays). While it is a fuzzy as it was a few years back (I'm little late on that 15 days), I've had a [more recent confusion regarding transifex](https://github.com/MarcusWolschon/osmeditor4android/issues/3068) which might help to show the confusion[^1]. 

In short, it seems you indeed need to choose _"join an existing organization"_, but then you **also** have to wait some undermined time (being confronted with confusion UI) until you are _"approved"_ manually by that organization, and only then will things start to work, and will look very confusing until such moment.

To someone used to just create an account and start contributing translations right away (e.g. Weblate, Poeditor, Codeberg translate, ...) it seems very confusing, especially as there are lacking **any indications** that your request to join the problem is currently pending approval, or even a list a project you've requested to join.

---

While bad UI/UX is Transifex's fault, iD docs should at least explain the joining workflow, i.e. describe the need to wait (perhaps with estimate for how long - e.g. _"usually within 1-3 days"_ or whatever) , and warn the user that UI would look broken and will be nonfunctional until such moment when they're approved. 

[^1]: although one should note that the IX confusion is likely (i.e. I don't remember details anymore) visually **different** if it is the first transifex project you're trying to help translate, or if you're already member of one (or more) projects and you're trying to join another one.

**Comment by tyrasd** [comment]
The documentation was improved a bit in #11512 last year. Is there still more to improve?

**Comment by matkoniecz** [comment]
I tested new steps with test account and it seems to avoid the trap described here.

**Comment by matkoniecz** [comment]
Though we got a new trap, I opened PR to give advise.

**Comment by mnalis** [comment]
> The documentation was improved a bit in https://github.com/openstreetmap/iD/pull/11512 last year. Is there still more to improve?

Yes. While there is the initial problem about different signup paths and business email requirement (which https://github.com/openstreetmap/iD/pull/11512 and https://github.com/openstreetmap/iD/pull/12156 attempt to address),  there an **additional** problem explained in https://github.com/openstreetmap/iD/issues/9562#issuecomment-4048634648 (and its link), which is that:

-  when you do finish the sign up process you still **cannot do anything and UI is confusing**, ***until such time*** as some project admin approves you. That waiting period needs to be explained / warned about.

**Comment by matkoniecz** [comment]
added in https://github.com/openstreetmap/iD/pull/12156

> After registration your account must be manually approved, you are unable to translate until that happens.
